File: InMotionGIT.Common/DataType/LookUpPackage.vb
Imports System.Runtime.Serialization
Imports System.Xml.Serialization

Namespace DataType

    <DataContract(Namespace:="urn:InMotionGIT.Common.LookUpPackage")>
    <Serializable()>
    <XmlType(Namespace:="urn:InMotionGIT.Common.LookUpPackage")>
    <XmlRoot(Namespace:="urn:InMotionGIT.Common.LookUpPackage")>
    <Attributes.TypeStructure("Code")>
    Public Class LookUpPackage

        <DataMember()>
        <XmlAttribute()>
        Public Property Key As String

        <DataMember()>
        <XmlAttribute()>
        Public Property Items As List(Of InMotionGIT.Common.DataType.LookUpValue)

        <DataMember()>
        <XmlAttribute()>
        Public Property Count As Integer

        Public Sub New()
            MyBase.New()
        End Sub

    End Class

End Namespace

File: Common/InMotionGIT.Common.Domain/DataType/LookUpPackage.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace InMotionGIT.Common.Domain.DataType;


[DataContract(Namespace = "urn:InMotionGIT.Common.LookUpPackage")]
[Serializable()]
[XmlType(Namespace = "urn:InMotionGIT.Common.LookUpPackage")]
[XmlRoot(Namespace = "urn:InMotionGIT.Common.LookUpPackage")]
[Attributes.TypeStructure("Code")]
public class LookUpPackage
{

    [DataMember()]
    [XmlAttribute()]
    public string Key { get; set; }

    [DataMember()]
    [XmlAttribute()]
    public List<LookUpValue> Items { get; set; }

    [DataMember()]
    [XmlAttribute()]
    public int Count { get; set; }

    public LookUpPackage() : base()
    {
    }

}